composer.json

```
@@ -13,7 +13,7 @@
     "require": {
         "silverstripe/framework": "^4",
         "silverstripe/cms": "^4",
-        "guzzlehttp/guzzle": "^7.3",
+        "guzzlehttp/guzzle": "^6.5 || ^7.3",
```

This won't satisfy one part of the problem per the PR description:
> silverstripe/textextraction 3.3.x-dev requires guzzlehttp/guzzle ~6.3.0
```suggestion
        "guzzlehttp/guzzle": "^6.3 || ^7.3",
```

---

Updated